Src/MoneyFox.Windows/Views/UserControls/SettingsGeneralUserControl.xaml.cs

```diff
@@ -1,10 +1,44 @@
-﻿namespace MoneyFox.Windows.Views.UserControls
+﻿
+namespace MoneyFox.Windows.Views.UserControls
 {
     public sealed partial class SettingsGeneralUserControl
     {
+        public global::Windows.Storage.ApplicationDataContainer localSettings = global::Windows.Storage.ApplicationData.Current.LocalSettings;
+        public global::Windows.Storage.StorageFolder localFolder = global::Windows.Storage.ApplicationData.Current.LocalFolder;
+
         public SettingsGeneralUserControl()
         {
             InitializeComponent();
         }
+
+        private void default_language_box_SelectionChanged(object sender, global::Windows.UI.Xaml.Controls.SelectionChangedEventArgs e)
```

Comment:
This should be done in the ViewModel so that it can be reused on other plattforms as Android.